db/imgcodec/jpeg/generate.py

```diff
@@ -0,0 +1,37 @@
+#!/bin/python3
```

Comment:
Do we need this script at all? I don't know what was the approach in corresponding PRs, could you elaborate?

Reply:
It is there to have some info about how are the references generated.